pycalphad/core/compiled_model.pyx

```diff
@@ -642,14 +638,14 @@ cdef public class CompiledModel(object)[type CompiledModelType, object CompiledM
                 out[out_idx] += disordered_energy
         if self._debug:
             debugout = np.zeros_like(out)
-            self._debug_energy(debugout, np.asfortranarray(dof), np.ascontiguousarray(parameters))
+            self._debug_energy(debugout, np.ascontiguousarray(dof), np.ascontiguousarray(parameters))
```

Comment:
No, it only matters for the energy function because the evaluation function is vectorized, so the memory layout in 2-D matters. For the gradient, everything is 1-D so the memory layouts are equivalent. For the Hessian, we should change it (since it's symmetric, the memory layout doesn't technically matter, which it why it still works)